src/Manager.php

```diff
@@ -65,12 +65,12 @@ public function repository()
     /**
      * Return Table
      *
-     * @return \Cake\ORM\Table Table Instance
+     * @return \Cake\Datasource\RepositoryInterface Repository Instance
```

Comment:
This does change the return and it's not BC per-se, but I'm going to merge this in as it's the proper way to do things.